azure-pipelines.yml

```diff
@@ -38,13 +38,13 @@ stages:
 
       dependencies:
         - source: '\\nirvana\Measurements\VeriStandAddons\scan_engine_fxp'
-          file: 'FXP.llb'
+          file: '$target\FXP.llb'
```

Comment:
will not work until https://github.com/ni/niveristand-custom-device-build-tools/pull/163 is submitted.

Reply:
merged